cli/pcluster/cfnconfig.py

```diff
@@ -381,14 +381,27 @@ def __check_account_capacity(self):
 
             test_ami_id = self.__get_latest_alinux_ami_id()
 
-            ec2.run_instances(
-                InstanceType=instance_type,
-                MinCount=max_size,
-                MaxCount=max_size,
-                ImageId=test_ami_id,
-                SubnetId=subnet_id,
-                DryRun=True,
-            )
+            if self.parameters.get("PlacementGroup"):
+                ec2.run_instances(
+                    InstanceType=instance_type,
+                    MinCount=max_size,
+                    MaxCount=max_size,
+                    ImageId=test_ami_id,
+                    SubnetId=subnet_id,
+                    Placement={
```

Comment:
Is it possible to avoid the if else block? With something like:
```py
Placement={"GroupName": self.parameters.get("PlacementGroup")} if self.parameters.get("PlacementGroup") else {}
```

Reply:
works brilliantly. Thanks @enrico-usai !